apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.resolver.ts

```diff
@@ -33,7 +33,25 @@ export class RecyclingRequestResolver {
     return res
   }
 
-  @Authorize({ roles: [Role.developer, Role.recyclingFund] })
+  @Query(() => [RecyclingRequestModel])
+  async skilavottordUserRecyclingRequest(
+    @CurrentUser() user: User,
+    @Args('permno') perm: string,
+  ): Promise<RecyclingRequestModel[]> {
+    const res = await this.recyclingRequestService.findRecyclingRequestWithPermno(
```

Comment:
Why the logging?


The name `findRecyclingRequestWithPermno` doesn't imply that this is for a single user.